hassio/src/hassio-main.ts

```diff
@@ -94,7 +95,10 @@ class HassioMain extends ProvideHassLitMixin(HassRouterPage) {
     applyThemesOnElement(
```

Comment:
Does this function also need some version check or is that ok?

Reply:
We have always used that function, so I think this is fine.
This is tested with 1.110 and 114 dev

Reply:
But that function was changed pretty heavily, and checks for properties in hass.selectedTheme

Reply:
True, but that does not seem to matter.

Tested both versions with:
- Custom theme
- Backend selected
- Default

In all cases this now does what it's expected of it.